charts/shoot-core/components/charts/cluster-identity

```diff
@@ -1 +0,0 @@
-../../../cluster-identity
```

Comment:
This will cause grm to delete the `cluster-identity` configmap from the `shoot-core` MR and then recreate it again for the `cluster-identity` MR, right?
Do you see any problem with this?

Reply:
I assumed it should be fine, it's only gone for a minute or so and will then re-appear.

Reply:
Ok good 👍